Polyhedron/demo/Polyhedron/Plugins/Surface_mesh_deformation/Edit_polyhedron_plugin.cpp

```diff
@@ -457,6 +459,13 @@ Polyhedron_demo_edit_polyhedron_plugin::convert_to_plain_polyhedron(Item_id i,
   Scene_polyhedron_item* poly_item = edit_item->to_polyhedron_item();
   scene->replaceItem(i, poly_item);
   delete edit_item;
+  if(saved_color.size() >0)
```

Comment:
Quickly reading the code of the draw, it seems useless to save the color vector since calling `invalidateOpenGLBuffers()` will trigger a call to `compute_normals_and_vertices()` that will fill (overwrite) that vector. Simply drop the local color vector and the patch should be sufficient.